packages/core/src/errors.js

```diff
@@ -16,10 +16,10 @@ function createErrorType(name, init, superErrorClass = Error) {
             return `${this.name}: ${this.message}`;
         };
     }
-    E.prototype = new superErrorClass();
-    E.prototype.name = name;
-    E.prototype.constructor = E;
-    return E;
+    JsJodaException.prototype = Object.create(superErrorClass.prototype);
```

Comment:
Previously a full instance of the superClass was created just to inherit from it, but a simple object with the right prototype is enough.